### ISSUE TITLE
Server Error

### ISSUE THREAD

**Issue by Skokan44** [fitak/ZUMScore#10]
Nevím v čem je problém, ale když se snažím nahrát svoje řešení, tak server hlásí chybu.

**Comment by janzal** [comment]
No... Tak tam budes mit asi chybu, me to funguje ;) Muzes sem dat treba tvuj seznam uzlu nebo tak neco?

**Comment by Skokan44** [comment]
Tak jsem zkusil ten seznam uzlů znovu naparsovat a commitnout pomocí knihovny a to se mi povedlo, nicméně manuálně mi to stejně pořád hlásí chybu serveru. 

Seznam uzlů zde:
[best.txt](https://github.com/fitak/ZUMScore/files/997836/best.txt)
